service/create/service.go

```diff
@@ -462,7 +463,7 @@ func validateIDs(ids []string) bool {
 
 func (s *Service) addFunc(obj interface{}) {
 	cluster := *obj.(*awstpr.CustomObject)
-	s.logger.Log("info", fmt.Sprintf("creating cluster '%s'", cluster.Name))
+	s.logger.Log("info", fmt.Sprintf("creating cluster '%s'", key.ClusterID(cluster)))
```

Comment:
Cool you catched this. I was thinking about improving the logging situation with a logging resource wit the reconciliation framework. Need to scribble a bit. This would prevent repeating all the same bits over and over again and logs in the resources only contain the necessary bits.